src/ProjectList/ProjectCardList.tsx

```diff
@@ -1,18 +1,23 @@
 import { Suspense } from "react";
 
 import { EmptyCard } from "../@shared";
-import { useAuth, useProjectListQuery } from "../@hooks";
+import { useUserQuery } from "../@hooks";
 import { ProjectCard } from "./ProjectCard";
+import { NotFoundError } from "../@utils/createError";
+import { ERROR } from "../@config/message";
 import * as css from "./ProjectCardList.css";
 
 export function ProjectCardList({ searchword }: { searchword: string }) {
-  const { user } = useAuth();
-  const { data: projects } = useProjectListQuery();
+  const { data: user } = useUserQuery();
+
+  if (!user) {
+    throw new NotFoundError(ERROR.NOT_FOUND.PROJECT_NAME);
+  }
 
   return (
     <section className={css.container}>
-      {projects && projects.length ? (
-        projects
+      {user.projects && user.projects.length ? (
```

Comment:
```suggestion
      {user?.projects.length ? (
```

optional chaining을 쓰면 더 간결해질 것 같아요!